main.go

```diff
@@ -109,40 +109,69 @@ func getTopNodes(projectID, token string) []FigmaNode {
 	return topNodes
 }
 
+func mergeMap(maps []map[string]string) map[string]string {
+	result := make(map[string]string)
+	for _, m := range maps {
+		for k, v := range m {
+			result[k] = v
+		}
+	}
+	return result
+}
+
+func chunkBy(items []string, chunkSize int) (chunks [][]string) {
+	for chunkSize < len(items) {
+		items, chunks = items[chunkSize:], append(chunks, items[0:chunkSize:chunkSize])
+	}
+
+	return append(chunks, items)
+}
+
 func getExportedURLs(projectID string, token string, nodeIDs []string) map[string]string {
-	params := url.Values{}
-	params.Set("ids", strings.Join(nodeIDs, ","))
-	params.Set("format", extension)
 
-	uri := filepath.Join(
-		host, version, "images", projectID,
-	)
-	uri = fmt.Sprintf("https://%s?%s", uri, params.Encode())
+	nodeIdChunks := chunkBy(nodeIDs, 20)
 
-	client := new(http.Client)
-	req, err := http.NewRequest("GET", uri, nil)
-	if err != nil {
-		log.Fatalf("failed to initialize http instance: %+v", err)
-	}
-	req.Header.Set("X-FIGMA-TOKEN", token)
+	urlMaps := make([]map[string]string, len(nodeIdChunks), len(nodeIdChunks))
 
-	resp, err := client.Do(req)
-	if err != nil {
-		log.Fatalf("failed to http request: %+v", err)
-	}
-	defer resp.Body.Close()
+	for _, chunk := range nodeIdChunks {
+		params := url.Values{}
+		params.Set("ids", strings.Join(chunk, ","))
+		params.Set("format", extension)
 
-	bodyText, err := ioutil.ReadAll(resp.Body)
-	if err != nil {
-		log.Fatalf("failed to io read dir: %+v", err)
-	}
+		uri := filepath.Join(
+			host, version, "images", projectID,
+		)
+		uri = fmt.Sprintf("https://%s?%s", uri, params.Encode())
 
-	var decoded ImagesResponse
-	if err = json.Unmarshal(bodyText, &decoded); err != nil {
-		log.Fatalf("failed to json unmarshal: %+v", err)
+		client := new(http.Client)
+		req, err := http.NewRequest("GET", uri, nil)
+		if err != nil {
+			log.Fatalf("failed to initialize http instance: %+v", err)
+		}
+		req.Header.Set("X-FIGMA-TOKEN", token)
+
+		resp, err := client.Do(req)
```

Comment:
forで分回す場合はclient使い回した方が良さそうなので
```suggestion
		resp, err := http.DefaultClient.Do(req)
```
net/httpパッケージのデフォルトの使った方がいいかも

Reply:
client を上に上げるのと何か違うんですか？

Reply:
毎回clientのインスタンス化しなくて良くて省メモリ

Reply:
あぁ、上にあげるってforの外に出すってことか
httpパッケージに同じものがあるのでわざわざ新規で立てなくて良さそうだなと思った

Reply:
なるほど
まぁ同じものがあるなら DefaultClient で良さそうですね

Reply:
小ネタですがコネクションが再利用されたりもします

Reply:
https://github.com/nl-plus-doc/figma-exporter/pull/16/commits/47f5d909ce6b91fe0e7d1085fc08cd886bd249e1 で修正